GDJS/Runtime/SceneLayer.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * GDevelop JS Platform
+ * Copyright 2013-2016 Florian Rival (Florian.Rival@gmail.com). All rights reserved.
+ * This project is released under the MIT License.
+ */
+namespace gdjs {
+  /**
+   * Represents a layer of a container, used to display objects.
```

Comment:
Update this comment, to explain that it's a scene layer and why it's different from a gdjs.Layer